src/librustc_target/spec/i686_uwp_windows_msvc.rs

```diff
@@ -0,0 +1,26 @@
+use crate::spec::{LinkerFlavor, Target, TargetResult};
+
+pub fn target() -> TargetResult {
+    let mut base = super::windows_uwp_msvc_base::opts();
+    base.cpu = "pentium4".to_string();
+    base.max_atomic_width = Some(64);
+    base.has_elf_tls = true;
+    base.pre_link_args
+        .get_mut(&LinkerFlavor::Msvc).unwrap()
+        .push("/LIBPATH:C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\lib\\store"
```

Comment:
Hm so FWIW I didn't review these files too closely before, but this is fishy in that it should be handled by `cc` not by linker flags here, because this path is guaranteed to change over time and not be correct for everyone's system.

Reply:
I can't say that I like that too much, but AFAIU the author didn't manage to work something else out. I'll check with him but I'm fine if this config only comes at a later time.